src/pages/interim/affiliate/_what-to-know.js

```diff
@@ -23,7 +23,7 @@ const WhatToKnow = () => {
                 </Text>
                 <Text align="center" size="var(--text-size-m)">
                     <Localize
-                        translate_text="The new affiliate login URL will change to <0>login.deriv.com</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>."
+                        translate_text={`The new affiliate login URL will change to <0>${login_url}</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>.`}
```

Comment:
I think you have to use `values` here, e.g. `values={{ login_url }}` then place `<0>{{login_url}}</0>`

Reply:
Hi @sean-binary , do you have any idea, from where I can view this page ?

Reply:
I'll pm you.